### PR TITLE
OCPBUGS-41969: OCPBUGS-42352: add service weight

### PR DESCRIPTION
**Pull request by upalatucci**
Added service weight

small fix: change `SERVICE_FIELD_ID` value;

**Before**

Screenshot 2024-09-30 at 11 18 14

**After**

Screenshot 2024-09-30 at 11 17 44

### CODE REVIEW

File: src/views/routes/form/ServiceSelector.tsx [hunk 1]
```diff
@@ -55,44 +56,71 @@ const ServiceSelector: FC<ServiceSelectorProps> = ({ namespace }) => {
...
-              <HelperText>
-                <HelperTextItem>{t('Service to route to.')}</HelperTextItem>
-              </HelperText>
-            </FormHelperText>
-          </FormGroup>
+              <FormHelperText>
+                <HelperText>
+                  <HelperTextItem>{t('Service to route to.')}</HelperTextItem>
+                </HelperText>
+              </FormHelperText>
```
Comment (avivtur) [inline src/views/routes/form/ServiceSelector.tsx:97]: Please use FormGroupHelperText component from utils instead

File: src/views/routes/form/ServiceSelector.tsx [hunk 2]
```diff
@@ -55,44 +56,71 @@ const ServiceSelector: FC<ServiceSelectorProps> = ({ namespace }) => {
...
+              />
+              <FormHelperText>
+                <HelperText>
+                  <HelperTextItem>
+                    {t(
+                      'A number between 0 and 255 that depicts relative weight compared with other targets.',
+                    )}
+                  </HelperTextItem>
+                </HelperText>
+              </FormHelperText>
```
Comment (avivtur) [inline src/views/routes/form/ServiceSelector.tsx:121]: Here too :)

File: src/views/routes/form/ServiceSelector.tsx [hunk 3]
```diff
@@ -55,44 +56,71 @@ const ServiceSelector: FC<ServiceSelectorProps> = ({ namespace }) => {
...
+              <FormHelperText>
+                <HelperText>
+                  <HelperTextItem>{t('Service to route to.')}</HelperTextItem>
+                </HelperText>
+              </FormHelperText>
+            </FormGroup>
+
+            <FormGroup fieldId={SERVICE_WEIGHT_FIELD_ID} label={t('Service weight')}>
+              <TextInput
+                defaultValue={100}
```
Comment (avivtur) [inline src/views/routes/form/ServiceSelector.tsx:102]: why do we want to give this arbitrary default value? if there's a good reason, add this to a const with a meaningful name

Comment (upalatucci) [reply src/views/routes/form/ServiceSelector.tsx:102]: okay. if this field is not defined, the default value is 100. this is the reason. I'll add a const

File: start-console.sh [hunk 4]
```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 #Please run this script under bash 4+
-set -euo pipefail
+# set -euo pipefail
```
Comment (avivtur) [inline start-console.sh:3]: Why commenting this?

**Review by avivtur — APPROVED** [review]

**Pull request merged**